mmcv/ops/csrc/pytorch/nms.cpp

```diff
@@ -69,6 +78,12 @@ Tensor nms(Tensor boxes, Tensor scores, float iou_threshold, int offset) {
     return nms_cuda(boxes, scores, iou_threshold, offset);
 #else
     AT_ERROR("nms is not compiled with GPU support");
+#endif
+#ifdef MMCV_WITH_MLU
+  } else if (boxes.device().type() == at::kMLU) {
+    CHECK_MLU_INPUT(boxes);
```

Comment:
`CHECK_MLU_INPUT` is not defined.

Reply:
I added it in `mmcv/ops/csrc/common/pytorch_cpp_helper.hpp` and push that branch.